src/dispatchers.ts

```diff
@@ -161,10 +161,12 @@ export async function handleCreate(
         return;
     }
 
-    // TODO Check Sender is in our following
     const createJson = await create.toJsonLd();
     ctx.data.globaldb.set([create.id.href], createJson);
-    await addToList(ctx.data.db, ['inbox'], create.id.href);
+    const following = await ctx.data.db.get<string[]>(['following']);
+    if (following?.includes(sender.id.href)) {
```

Comment:
Not a blocker - We have a [helper](https://github.com/TryGhost/ActivityPub/blob/main/src/helpers/activitypub/actor.ts#L78) for this